README.md

```diff
@@ -15,136 +15,71 @@ When you have set up your repo
 
-## How to install and use this template
-
-### Install with Git
-
-1. Click the green button `Use this template`.
-
-1. Give a name to the repository you want to create. Something short that
-   contains the name of your experiment: `analysis_my_study`.
-
-1. Decide if you want this new repo to be public or private.
-
-1. Click on `Create repository from template`
-
-You now have a copy of the template on your Github account. You can then
-download the code and the pre-set dependencies like this.
-
-1. Click on green `Download` button and copy the `URL_to_your_repo` that is
-   shown there.
-
-1. Open a terminal and type this:
-
-```bash
-git clone --recurse-submodules URL_to_your_repo
-```
+This is a set of functions to prepare a source fMRI dataset for BIDSification and analyses
 
-This will set up everything automatically in your current directory.
+This can perform:
 
-## Content
+- 3D to 4D nii files conversion
 
-```bash
-├── .git
-│   ├── COMMIT_EDITMSG
-│   ├── FETCH_HEAD
-│   ├── HEAD
-│   ├── ORIG_HEAD
-│   ├── branches
-│   ├── config
-│   ├── description
-│   ├── hooks
-│   │   ├── pre-commit.sample
-│   │   └── pre-push.sample
-│   ├── ...
-│   └── ...
-├── .github                # where you put anything github related
-│   └── workflows          # where you define your github actions
-│       └── moxunit.yml    # a yaml file that defines a github action
-├── lib                    # where you put the code from external libraries (mathworks website or other github repositories)
-│   └── README.md
-├── src                  # WHERE YOU PUT YOUR CODE
-│   ├── README.md
-│   └── miss_hit.cfg
-├── docs                 # where you write your documentation using sphinx
-│   ├── make.bat
-│   ├── Makefile
-│   ├── README.md
-│   ├── requirements.txt # to set up the python environment for the doc
-│   └── source           # the actual reStructured text file for your doc
-│       ├── conf.py
-│       ├── function_description.rst
-│       ├── index.rst
-│       ├── _static
-│       └── _templates
-├── tests               # where you put your unit tests
-|   ├── README.md
-|   └── miss_hit.cfg
-├── .travis.yml         # where you define the continuous integration done by Travis
-├── LICENSE
-├── README.md
-├── miss_hit.cfg        # configuration file for the matlab miss hit linter
-└── initEnv.m           # a .m file to set up your project (adds the right folder to the path)
-```
-
-## Keeping your code stylish: miss hit linter
+- anatomical defacing [ W I P ]
 
-### Matlab code style guide and quality
+- gzip the nii files in a BIDS raw folder
 
-We use the [MISS_HIT linter](https://github.com/florianschanda/miss_hit/) to
-automatically enforce / fix some code style issues and check for code quality.
+The core functions are in the `src` folder. This set of functions is intended to be added manually
+as a submodule to the analysis repo forked from
+[template_fMRI_analysis](https://github.com/cpp-lln-lab/template_fMRI_analysis)
```

Comment:
Do you think it would be simpler to add this as a submodule to the template repo by default?

No need for a final answer now.

Something to keep in mind.

Reply:
For our workflow yes, indeed.  But so far it is not clear to me if it could be of general interest for data coming from a different machine.  So for the moment, I would keep as a possible "extensions".

Reply:
Agree !!